### PR TITLE
Remove identifiers from integration tests

### PR DESCRIPTION
**Pull request by asparuhft**
# Description

## What

Identifiers are cleaned up in the integration tests although they are not created.

## Why

JIRA Ticket [UPPSF-2222](https://financialtimes.atlassian.net/browse/UPPSF-2222)

## Scope and particulars of this PR (Please tick all that apply)

- [X] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: relations/fixtures/Content-3fc9fe3e-af8c-1a1a-961a-e5065392bb31.json [hunk 1]
```diff
@@ -1,13 +1,6 @@
 {
   "uuid": "3fc9fe3e-af8c-1a1a-961a-e5065392bb31",
   "title": "Awesome story with no annotations",
-  "byline": "By Matthew Garrahan in Los Angeles",
```
Comment (mchompalova) [inline relations/fixtures/Content-3fc9fe3e-af8c-1a1a-961a-e5065392bb31.json:4]: Why do we remove byline here?

**Review by mchompalova — CHANGES_REQUESTED** [review]

File: relations/fixtures/Content-3fc9fe3e-af8c-1a1a-961a-e5065392bb31.json (hunk 1 above)
Comment (asparuhft) [reply relations/fixtures/Content-3fc9fe3e-af8c-1a1a-961a-e5065392bb31.json:4]: Just discussed this topic with Kalin. I will revert the changes on the content fixtures in all of the PRs I have raised.

**Review by mchompalova — COMMENTED** [review]
I'm a bit confused. The description says: Identifiers are wrongly used in the content text fixtures because the content-rw-neo4j is not supporting them.
And still we leave them in the fixtures?

**Comment by asparuhft** [comment]
The content-rw-neo4j is ignoring parameters like identifiers, byline, brands. Furthermore, these identifiers are semantically different from the ones created by concepts-rw-neo4j service. I just decided to clean up these parameters (identifiers, byline, ...) from the content fixtures as they are not used. However, Kalin proposed that we keep them as these are more close to the ones that the content-rw-neo4j gets from Kafka through the ingester. These makes sense so I reverted the changes on the content fixtures. 
What is important in this PR is that we are trying to delete Identifiers in the cleanup part of the integration tests, which is actually not needed because no Identifiers are created.

**Comment by mchompalova** [comment]
> The content-rw-neo4j is ignoring parameters like identifiers, byline, brands. Furthermore, these identifiers are semantically different from the ones created by concepts-rw-neo4j service. I just decided to clean up these parameters (identifiers, byline, ...) from the content fixtures as they are not used. However, Kalin proposed that we keep them as these are more close to the ones that the content-rw-neo4j gets from Kafka through the ingester. These makes sense so I reverted the changes on the content fixtures.
> What is important in this PR is that we are trying to delete Identifiers in the cleanup part of the integration tests, which is actually not needed because no Identifiers are created.

Thank you for the detailed explanation. Can you squash the two commits into one?

**Review by karsov — APPROVED** [review]

**Review by mchompalova — APPROVED** [review]

**Review by karsov — APPROVED** [review]

**Pull request merged**